xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3077,6 +3077,8 @@ platform.livetable.paginationResultsSingle=Result <span class="currentResultsNo"
 platform.livetable.paginationResultsMany=Results <span class="currentResultsNo">{0} - {1}</span> of <span class="totalResultsNo">{2}</span>
 platform.livetable.paginationResults=Results
 platform.livetable.paginationResultsOf=out of
+platform.livetable.false=No
+platform.livetable.true=Yes
```

Comment:
we already have generic keys for this which might be better to use. For example:

    no_0=No
    yesno_1=Yes